### PR TITLE
feat(dh): basic notification banner and panel items

### PR DESCRIPTION
**Pull request by dzhavat**
### DataHub
- basic notification center 
- basic notification banner
- add "hot-toast" package
- mock "hot-toast" package in tests

Note: The `dh-hot-toast.mock.ts` file is at root level in this PR. There will be a follow up PR to move it in a proper place. Maybe together with other test util mocks. 

Panel items

image

Banner

image

## References
- #0000

### CODE REVIEW

File: libs/dh/core/shell/src/lib/notification-center/dh-notifications-center.component.ts [hunk 1]
```diff
@@ -91,4 +133,30 @@ export class DhNotificationsCenterComponent {
...
+  showNotificationBanner(): void {
+    this.hotToast.show(DhNotificationBannerComponent, {
+      position: 'top-right',
+      dismissible: true,
+      autoClose: false,
+      style: {
+        'background-color': 'rgba(219, 219, 219, 0.3)',
+        border: '1px solid #bdbdbd', // gray-400
+        width: '345px',
+        color: 'rgba(0, 0, 0, 0.87)', // on-light-high-emphasis
```
Comment (mimse) [inline libs/dh/core/shell/src/lib/notification-center/dh-notifications-center.component.ts:146]: Comments: Seems abit od to provide styles like this, is there an other way?

**Comment by mimse** [comment]
Praise: Looks good, just a few comments 🥇

File: libs/dh/core/shell/src/lib/notification-center/dh-notifications-center.component.ts (hunk 1 above)
Comment (dzhavat) [reply libs/dh/core/shell/src/lib/notification-center/dh-notifications-center.component.ts]: Found out that most of the properties can be moved to a CSS variable. Those that couldn't are kept here. Any existing Watt colors are now taken from a service instead

File: apps/dh/app-dh/src/styles.scss [hunk 2]
```diff
@@ -17,6 +17,17 @@
...
 @use "highlight.js/styles/github.css";
+@use "@ngxpert/hot-toast/src/styles/styles.scss";
+
+:root {
+  --hot-toast-message-margin: 0;
+  --hot-toast-message-justify-content: flex-start;
+
+  --hot-toast-close-btn-background-size: 8px;
+
+  --hot-toast-padding: 16px 14px 16px 24px;
```
[the remark this answers is not in the document]
Comment (dzhavat) [reply apps/dh/app-dh/src/styles.scss]: Should be fixed now.

File: libs/dh/core/shell/src/lib/notification-center/dh-notifications-center.component.ts (hunk 1 above)
Comment (mimse) [reply libs/dh/core/shell/src/lib/notification-center/dh-notifications-center.component.ts]: Perfect

File: apps/dh/app-dh/src/styles.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (mimse) [reply apps/dh/app-dh/src/styles.scss]: Nice

**Pull request merged**